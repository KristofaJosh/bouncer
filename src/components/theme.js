import * as colors from "../components/Colors/color";
const theme = {
  colors,
};
export default theme;