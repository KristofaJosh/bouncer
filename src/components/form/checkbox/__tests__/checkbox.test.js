import React from 'react'
import renderer from 'react-test-renderer'
import {ThemeProvider} from 'styled-components'
import CheckGroup from '../index'
import theme from '../../../theme'


it('renders correctly', ()=>{
    const tree = renderer.create(
    <ThemeProvider theme={theme}>
      <CheckGroup/>  
    </ThemeProvider>
    
    
    ).toJSON();
    expect(tree).toMatchSnapshot();
})
