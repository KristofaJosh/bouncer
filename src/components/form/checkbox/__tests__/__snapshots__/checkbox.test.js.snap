// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa fBEhyh"
  onClick={[Function]}
>
  <input
    checked={false}
    className="sc-htpNat eUTAqn"
    type="checkbox"
  />
  <div
    checked={false}
    className="sc-bxivhb bCOqAG"
  >
    <svg
      className="sc-bwzfXH bkYyAD"
      viewBox="0 0 24 24"
    >
      <polyline
        points="20 6 9 17 4 12"
      />
    </svg>
  </div>
</div>
`;
